pom.xml

```diff
@@ -1,36 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
wollten wir nicht alles jetzt mit einem Maven-Multi-Modul-Projekt machen?

Reply:
Yes but we had to undo it in this case as the Maven Central Release wasn't working as intended.
In this case that was specific to our pom structure.
I think the actions should support both 🤔